packages/schemas/src/gen/index.ts

```diff
@@ -43,8 +45,7 @@ const generate = async () => {
             const name = normalizeWhitespaces(prefix).split(' ')[2];
             assert(name, 'Missing table name: ' + prefix);
 
-            const fields = removeParentheses(body)
-              .split(',')
+            const fields = splitAtCommasOutsideParentheses(body)
```

Comment:
Is there any reason to do this in the proposed way?

Reply:
It's the `oidc_model_instances` table definition. 

```sql
create table oidc_model_instances (
  model_name varchar(64) not null,
  id varchar(128) not null,
  payload jsonb /* @use OidcModelInstancePayload */ not null,
  expires_at timestamptz not null,
  consumed_at timestamptz,
  primary key (model_name, id)
);
```

After extracting the definitions of columns and constraints (using `findFirstParentheses`), we need to parse:

```ts
  model_name varchar(64) not null,
  id varchar(128) not null,
  payload jsonb /* @use OidcModelInstancePayload */ not null,
  expires_at timestamptz not null,
  consumed_at timestamptz,
  primary key (model_name, id)
```

We used to remove the parentheses and the content inside them 
first, and then split the column and the constraint definitions by `,`.

Since we expect to get the length limit from `varchar(n)`, we need to keep the parentheses and the content inside.

It's safe and precise to split them by the commas that are not in the parentheses.

Reference: [PostgreSQL DDL (Data Definition Language) `create table` statement](https://www.postgresql.org/docs/current/sql-createtable.html)

Reply:
Hmmm, I think the logic is valid. However, I might be nit-picking, I think the function name can be slightly refined, name it by what it does, not how it does it.
How about `splitColumnDefinitions`.

Reply:
I prefer to keep `splitAtCommasOutsideParentheses` a pure utility function and keep its name to avoid relating to the specific case such as `splitColumnDefinitions`.

Maybe I'd better add a comment to explain why use `splitAtCommasOutsideParentheses` here?